pkg/apis/autoscaling/v2beta1/conversion.go

```diff
@@ -299,7 +299,6 @@ func Convert_autoscaling_HorizontalPodAutoscaler_To_v2beta1_HorizontalPodAutosca
 		}
 		// copy before mutating
 		if !copiedAnnotations {
-			copiedAnnotations = true
```

Comment:
dropping this makes it hard to reason about future changes in the method that could check this... I'd probably add a comment exempting this

Reply:
Ack.

Reply:
Added lint ignore